.github/workflows/sonar.yaml

```diff
@@ -34,3 +34,34 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: ./gradlew build sonarqube --info
+  build-maven:
```

Comment:
Does this mean we now have two Sonar checks? If we feel confident, maybe we can just switch to Maven, this is not like a release job that we want to be more careful.

Reply:
It does. I was debating between removing the gradle version and keeping it -- I figured it would be harmless (just email spam) with two sonar jobs.

I'm pretty confident that we can remove the gradle version if we just want to have only one sonar check.